### PR TITLE
ci: fix labeller permissions

### PR DESCRIPTION
**Pull request by dabeeeenster**
Hit this error: https://github.com/Flagsmith/flagsmith/actions/runs/5535805708/jobs/10102641881?pr=2427
Fix as per https://github.com/actions/labeler#permissions

### CODE REVIEW

File: .github/workflows/github-labeler.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 name: Github Labeler
 
 on:
-  pull_request:
-    types: [opened, synchronize, reopened, ready_for_review]
+  - pull_request_target
```
Comment (matthewelwell) [inline .github/workflows/github-labeler.yml]: I think this syntax is wrong though? The change here should just be to add `_target` to line 4 and the rest should stay the same? 

image

**Comment by dabeeeenster** [comment]
The example given is:

image

I'm not actually sure why we are specifying those 4 particular PR types - any idea?

**Comment by dabeeeenster** [comment]
Updated

**Review by matthewelwell — APPROVED** [review]

**Comment by matthewelwell** [comment]
> The example given is:
> 
> image

Ah, sorry, YAML consistency is just baffling at times. 

> I'm not actually sure why we are specifying those 4 particular PR types - any idea?

Good question, I'm not sure. We probably should remove them and add the labels to all PRs I guess.

**Pull request merged**